notebooks/community/model_evaluation/automl_tabular_regression_model_evaluation.ipynb

```diff
@@ -0,0 +1,1567 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        TRAINING_JOB_DISPLAY_NAME = "train-pet-agefinder-automl"
</pre><p>Use UUID as part of the name</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>done</p>